.github/workflows/claude.yml

```
@@ -1,5 +1,78 @@
 name: Claude Code
 
+# Define reusable allowed tools list
+x-allowed-tools: &allowed-tools |
```

actionlint (1.7.7)

unexpected key "x-allowed-tools" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)